devolutions-agent/src/config.rs

```diff
@@ -275,10 +291,15 @@ pub mod dto {
 
         #[serde(skip_serializing_if = "Option::is_none")]
         pub remote_desktop: Option<RemoteDesktopConf>,
+
         /// Devolutions PEDM configuration
         #[serde(default, skip_serializing_if = "Option::is_none")]
         pub pedm: Option<PedmConf>,
 
+        /// Devolutions Session Host configuration
+        #[serde(default, skip_serializing_if = "Option::is_none")]
+        pub session_host: Option<SessionHostConf>,
```

Comment:
Naming is debatable - maybe someone has a better idea instead of "host"?

Reply:
Some suggestions:
- `now_host: NowHostConf`
- `now_dvc: NowDvcConf`
- `remote_desktop_host: RemoteDesktopHostConf`
- `remote_desktop_dvc: RemoteDesktopDvcConf`

By the way, is the name "Devolutions Host" already agreed upon?
Should it be "Devolutions Now" or "Devolutions Now Host" or "Devolutions Now Dvc", or something else?
(I don’t know if "Host" is a standard terminology for DVC servers.)